src/components/common/Content/TagContent.tsx

```diff
@@ -0,0 +1,56 @@
+import { css, Theme } from '@emotion/react';
+
+import { IconButton } from '../Button';
+import { labelCss } from '../styles';
+import Tag from '../Tag';
+
+export interface TagContentProps {
+  label?: string;
+  tags: TagType[];
+  /**
+   * 영감 추가일 경우, 추가 기능을 수행합니다.
+   *
+   * 영감 편집일 경우, 편집 기능을 수행합니다.
+   */
+  onEdit: VoidFunction;
+  /**
+   * 단일 Tag 클릭 할 경우, Action을 넘겨 줍니다.
+   */
+  onClickTag?: (tagId: number) => void;
+}
+
+export default function TagContent({ tags, label, onEdit, onClickTag }: TagContentProps) {
+  return (
+    <div css={tagContentWrapperCss}>
+      <span css={tagContentLabelCss}>{label ? label : '태그'}</span>
```

Comment:
jsx에서 삼항 연산자를 사용할 수도 있지만, 아래처럼 default 문을 사용해도 나쁘지 않을 거 같아요!! 👍 👍 👍 

```suggestion
export default function TagContent({ tags, label = '태그', onEdit, onClickTag }: TagContentProps) {
  return (
    <div css={tagContentWrapperCss}>
      <span css={tagContentLabelCss}>{label}</span>
```

Reply:
저도 디폴트를 넣어놓는 방식이나, 가독성 위해서 {label ?? '태그'} 이런 식으로도 작성하는 걸 선호한느 편입니다. :))

Reply:
적용완료했습니다! 3f6bce2